### PR TITLE
Document new tuple syntax.

### PR DESCRIPTION
**Pull request by varnerac**

### CODE REVIEW

**Review by lpil — APPROVED** [review]
Thank you. I'll hold off merging this until the new syntax is released

File: book-src/tour/tuples.md [hunk 1]
```diff
@@ -4,7 +4,13 @@ Lists are good for when we want a collection of one type, but sometimes we want
 to combine multiple values of different types. In this case tuples are a quick
 and convenient option.
 
+Gleam provides two ways to construct or match on tuples: the `#(1, 2, 3)` format,
+introduced in Gleam 0.14.5, and the original `tuple(1, 2, 3)` format.
```
Comment (tranzystorek-io) [inline book-src/tour/tuples.md:8]: Mind the release version, it's probably going to be 0.15.0

Comment (varnerac) [reply book-src/tour/tuples.md:8]: I'll fix.

**Comment by varnerac** [comment]
Meh, now I've created a merge conflict. Let me figure this out.

**Comment by lpil** [comment]
Don't worry I can fix that when I do all my merging on release!

**Comment by lpil** [comment]
I've merged this into my branch, thank you

**Pull request closed without merging**